client/src/redux/reducers/onboarding.reducers.js

```diff
@@ -1,6 +1,10 @@
 export const initialState = {
   admin: {
     users: [],
+    offset: 0,
+    limit: 25,
+    total: 0,
+    query: null,
```

Comment:
Get a warning in the searchbar for the null query field:

> Warning: `value` prop on `input` should not be null. Consider using an empty string to clear the component or `undefined` for uncontrolled components.

```suggestion
    query: '',
```